src/libraries/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_6_0/IdnaTest_6.txt

```diff
@@ -106,11 +106,11 @@ N;	a\u0628\u0308\u200C\u0308;	[B5 B6 C1];	[B5 B6 C1]
 #T;	A\u0628\u0308\u200C\u0308;	[B5 B6 C1];	[B5 B6]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
 N;	A\u0628\u0308\u200C\u0308;	[B5 B6 C1];	[B5 B6 C1]
 #T;	a\u0628\u0308\u200C\u0308\u0628b;	[B5];	[B5]	# GETASCII DOES NOT FAIL ON WINDOWS 8.1
-N;	a\u0628\u0308\u200C\u0308\u0628b;	[B5];	[B5]
+#N;	a\u0628\u0308\u200C\u0308\u0628b;	[B5];	[B5]
```

Comment:
>u0308 [](start = 12, length = 5)

any idea why these cases commented? I thought this used to run successfully before. no?

Reply:
Those were not running before because we were ignoring `NonTransitional` test cases. See: https://github.com/dotnet/runtime/pull/37281#discussion_r435562512